ddl/get.lua

```diff
@@ -111,6 +114,56 @@ local function get_space_schema(space_name)
     return space_ddl
 end
 
+local function prepare_sharding_func_for_call(space_name, sharding_func_def)
+    if type(sharding_func_def) == 'string' then
+        local sharding_func = utils.get_G_function(sharding_func_def)
+        if sharding_func ~= nil and
+           ddl_check.internal.is_callable(sharding_func) == true then
+            return sharding_func
+        end
+    end
+
+    if type(sharding_func_def) == 'table' then
+        local sharding_func = loadstring('return ' .. sharding_func_def.body)
+        if sharding_func == nil then
+            return nil, string.format(
+                "Body is incorrect in sharding_func for space (%s)", space_name)
+        end
```

Comment:
If something goes wrong, `loadstring()` returns an error in the second return value. It worth to concatenate it into our error.

Reply:
Added err to message:
```diff
--- a/ddl/get.lua
+++ b/ddl/get.lua
@@ -124,10 +124,10 @@ local function prepare_sharding_func_for_call(space_name, sharding_func_def)
     end
 
     if type(sharding_func_def) == 'table' then
-        local sharding_func = loadstring('return ' .. sharding_func_def.body)
+        local sharding_func, err = loadstring('return ' .. sharding_func_def.body)
         if sharding_func == nil then
             return nil, string.format(
-                "Body is incorrect in sharding_func for space (%s)", space_name)
+                "Body is incorrect in sharding_func for space (%s): %s", space_name, err)
         end
         return sharding_func()
     end
```